libs/constrained-generators/src/Constrained/Test.hs

```diff
@@ -152,6 +152,9 @@ tests =
     , testSpec "listSumPair" (listSumPair @Int)
     , testSpec "parallelLetPair" parallelLetPair
     , numberyTests
+    , testSpec "(And pair (x,y) test -- both (x <=. 5) and (y <=. 1)" andPair
+    , testSpec "(Or  pair (x,y) test -- at least one of (x <=. 5) or (y <=. 5)" orPair
+    , testSpec "size test" mapSizeConstrained
```

Comment:
remove this

Reply:
same as conversation above. Don't we want the baseline tests here?

Reply:
I think this is fine, if we change the `"size test"` to be `"mapSizeConstrained"` to ensure minimal risk of collision with the other PR.

Reply:
done